easybuild/easyblocks/a/allinea.py

```diff
@@ -103,10 +102,11 @@ def install_step(self):
             if path:
                 self.log.debug('system.config file %s found' % path)
             else:
-                raise EasyBuildError('No system.config file named %s found', sysconfig)
+                raise EasyBuildError('No system.config file named %s found', sysconf_path)
 
             copy_file(path, sysconf_path)
-            adjust_permissions(sysconf_path, stat.S_IRUSR|stat.S_IRGRP|stat.S_IROTH, recursive=False, relative=False)
+            adjust_permissions(sysconf_path, stat.S_IRUSR | stat.S_IRGRP |
+                               stat.S_IROTH, recursive=False, relative=False)
```

Comment:
I would use a more natural split here:

```suggestion
            adjust_permissions(sysconf_path, stat.S_IRUSR | stat.S_IRGRP | stat.S_IROTH
                               recursive=False, relative=False)